### PR TITLE
feat: adding transactions and fixing HasData issue

### PR DESCRIPTION
**Pull request by jleon15**
## Description

- Add transactions
- Fixes HasData issue

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

**Comment by djejaquino** [comment]
Fixes #69

File: README.md [hunk 1]
```diff
@@ -245,4 +240,5 @@ Each of these functions takes a value of the given basic type and returns a poin
 
 ## Author
 
-[Basis Theory](mailto:support@basistheory.com)
+
```
Comment (brigonzalez) [inline README.md:243]: need to revert a few things in this file

File: docs/ApplicationTemplatesApi.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ import (
     "context"
     "fmt"
     "os"
-    openapiclient "./openapi"
+    openapiclient "github.com/GIT_USER_ID/GIT_REPO_ID"
```
Comment (brigonzalez) [inline docs/ApplicationTemplatesApi.md:27]: these seem wrong as well. there are a few of these

**Review by brigonzalez — APPROVED** [review]

**Pull request merged**